Dockerfile

```diff
@@ -71,4 +71,4 @@ RUN mkdir /bzt-configs \
   && rm -rf /tmp/*
 
 WORKDIR /bzt-configs
-CMD bzt -l /tmp/artifacts/bzt.log /bzt-configs/*.yml
+ENTRYPOINT ["sh", "-c", "bzt /bzt-configs/*.yml -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
```

Comment:
Some explanation: the shell form of ENTRYPOINT [will ignore any `CMD` or `docker run` command line arguments](https://docs.docker.com/engine/reference/builder/#shell-form-entrypoint-example), so I switched to using the exec form. However, as they mention in [a note in this section of the documentation](https://docs.docker.com/engine/reference/builder/#exec-form-entrypoint-example):

> Unlike the shell form, the exec form does not invoke a command shell. This means that normal shell processing does not happen. For example, ENTRYPOINT [ "echo", "$HOME" ] will not do variable substitution on $HOME. If you want shell processing then either use the shell form or execute a shell directly, for example: ENTRYPOINT [ "sh", "-c", "echo $HOME" ]. When using the exec form and executing a shell directly, as in the case for the shell form, it is the shell that is doing the environment variable expansion, not docker.

So to continue using `*` to locate all `.yml` files, I started the ENTRYPOINT with `"sh", "-c"` to execute in a shell that will interpret `*` as a wildcard. In order to incorporate the additional command line arguments in the `sh` command as well, I ended it with a `"$@"`, which will expand to all positional parameters except $0. Since $0 is ignored, I included a throwaway parameter called `"ignored"` in the ENTRYPOINT so that all arguments to the `docker run` will be read.